instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/internal/HttpProtocolUtil.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.api.internal;
+
+import javax.annotation.Nullable;
+
+/**
+ * This class is internal and is hence not for public use. Its APIs are unstable and can change at
+ * any time.
+ */
+public final class HttpProtocolUtil {
+
+  public static String getProtocol(@Nullable String protocol) {
+    if (protocol != null && protocol.startsWith("HTTP/")) {
+      return "http";
+    }
+    return null;
+  }
+
+  public static String getVersion(@Nullable String protocol) {
+    if (protocol != null && protocol.startsWith("HTTP/")) {
```

Comment:
I extracted an util class because this same pattern is used in many instrumentations and all of them probably report http2 version as `2.0` instead of `2`.